packages/cli/src/commonlib/azureLogin.ts

```diff
@@ -235,8 +288,13 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
     return Promise.resolve(undefined);
   }
 
-  async getJsonObject(showDialog = true): Promise<Record<string, unknown> | undefined> {
-    const token = await AzureAccountManager.codeFlowInstance.getToken();
+  async getJsonObject(showDialog = true, tenantId = ""): Promise<Record<string, unknown> | undefined> {
```

Comment:
Is this a public function? Do not change contract of other public function(s).

Reply:
Fixed.